docs/guide/advocacy.md

```diff
@@ -0,0 +1,89 @@
+# Advocacy
+
+Much of e18e's work involves making changes in existing open source repositories.
+These repositories generally have their own workflows that their maintainers expect to adhere to.
+It's important when suggesting changes to repositories to respect those workflows and actively work with those maintainers.
+
+## Coordinating Issues
+
+Many repositories prefer discussing potential changes in an issue before they can fully review a pull request.
+Unless a repository maintainer has explicitly indicated a willingness to bypass issue practices, stick with their normal flow of filing an issue and fully filling out the most applicable issue template.
```

Comment:
just a minor wording suggestion as not all maintainers prefer an issue ahead of time
```suggestion
Unless a repository maintainer has explicitly indicated a willingness to bypass issue practices, stick with their normal flow and fully fill out the most applicable issue template if they prefer an issue to be filed ahead of any PRs.
```